src/main/resources/application.properties

```diff
@@ -1,8 +1,14 @@
+#DB-Free:
+spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration,\
+    org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration,\
+    org.springframework.boot.autoconfigure.jdbc.DataSourceTransactionManagerAutoConfiguration,\
+    org.springframework.boot.autoconfigure.data.web.SpringDataWebAutoConfiguration
+
```

Comment:
Is this duplicated? See the EnableAutoConfiguration annotation on FogbowManagerApplication.

Reply:
Yes it is, could you review this again?